packages/node_modules/@ciscospark/react-component-space-item/src/index.js

```diff
@@ -91,7 +94,8 @@ function SpaceItem({
     <div
       className={classNames('space-item', styles.item, {
         [styles.hasCallSupport]: !!hasCallSupport,
-        [styles.isDecrypting]: !!isDecrypting
+        [styles.isDecrypting]: !!isDecrypting,
+        [styles.active]: activeSpaceId === id
```

Comment:
I think it might make more sense to change this prop to `isActive` and do the `activeSpaceId` comparison in the spaces list.

Reply:
That works. Will update.